.dialyzer_ignore.exs

```diff
@@ -0,0 +1,3 @@
+[
+  {"", :unknown_type, 0} # GRPC 0.3 doesn't typecheck
```

Comment:
I've never seen this pattern before, but how does this only target grpc? I'd rather we didn't turn this off for the whole check

Reply:
yes it is a weird one, unfortunately I think we have to do this - the exact error I get from Dialyzer is this:

<img width="421" alt="image" src="https://user-images.githubusercontent.com/1388226/180437943-fdbdc6c6-47b5-4b54-b5a6-673de6c08841.png">

Apparently this can happen with generated code, so I am wondering if dialyzer is finding the error through some generated grpc code and thus not giving us a line number :(

I am hoping most dialyzer errors in the actual project should get line numbers so this won't match

Reply:
Can we just ignore grpc from dialyzer? I don't know if that is worse